LICENSE.md

```diff
@@ -0,0 +1,194 @@
+# LICENSE
+
+This repository contains multiple components licensed with different licenses.
+
+"Data" is licensed as CC BY-SA 4.0. "Code" is licensed as BSD-3-Clause. Specifically this means, that the subfolders
+
+- .devcontainer
+- .github
+- analytics
+- definitions
+- schlumpf
```

Comment:
nit: I'm not sure human beings can be licensed.

Reply:
I was more scared that Bavaria Media was going to complain. Maybe all of this is just a scam I'm doing to get fully permission to market the Smurf brand myself.